src/applications/facility-locator/components/ResultsList.jsx

```diff
@@ -1,14 +1,29 @@
 /* eslint-disable arrow-body-style */
 import React, { Component } from 'react';
+import _ from 'lodash';
 import PropTypes from 'prop-types';
 import { bindActionCreators } from 'redux';
 import { connect } from 'react-redux';
 import { updateSearchQuery, searchWithBounds } from '../actions';
 import LoadingIndicator from '@department-of-veterans-affairs/formation-react/LoadingIndicator';
 import SearchResult from './SearchResult';
 import Pagination from '@department-of-veterans-affairs/formation-react/Pagination';
+import { facilityTypes } from '../config';
 
 class ResultsList extends Component {
+  shouldComponentUpdate(nextProps) {
+    return (
+      nextProps.results !== this.props.results ||
```

Comment:
These are the only things that should trigger updates in this component. I need to search string to display the loading state but since it's updated on every input change, this prevents `ResultsList` from updating and triggering focus on every keypress.